go.mod

```diff
@@ -1,10 +0,0 @@
-module github.com/lucaslorentz/caddy-docker-proxy/v2
```

Comment:
Don't you still need the `go.mod` so others can install this as a plugin?

Reply:
Oh nevermind, scratch that. I see https://github.com/lucaslorentz/caddy-docker-proxy/blob/master/plugin/go.mod :sweat_smile:

Reply:
There is a go.mod inside plugin folder.